pkg/controller/virtualmachinevolumeexport/virtualmachinevolumeexport_controller.go

```diff
@@ -151,8 +151,10 @@ func (r *ReconcileVirtualMachineVolumeExport) validateVirtualMachineVolume() err
 		return goerrors.New("fail to get virtual machine volume")
 	}
 	// check if vmv is available
-	if vmVolume.Status.State != hc.VirtualMachineVolumeStateAvailable {
-		return goerrors.New("virtual machine volume is not available")
+	found, cond := util.GetConditionByType(vmVolume.Status.Conditions, hc.ConditionReadyToUse)
+	if !found || cond.Status != corev1.ConditionTrue {
```

Comment:
"!found" indicates that the condition does not exist. When the condition does not exist, it is the same when the condition state is unknown(corev1.ConditionUnknown) This mean the controller may not have determined its condition yet. How do you think it would be better to handle the case of '!found', 'true', 'false' and 'unknown'?

Reply:
I Agree. Thanks for your comment